jbmc/src/java_bytecode/select_pointer_type.h

```diff
@@ -36,6 +36,16 @@ class select_pointer_typet
 
 public:
   virtual ~select_pointer_typet() = default;
+
+  /// Select what type should be used for a given pointer type. For the base
+  /// class we just use the supplied type. Derived classes can override this
+  /// behaviour to provide more sophisticated type selection. Generic
```

Comment:
:pick: The coding standard says "Use American spelling for identifiers.", it doesn't say anything about spelling in comments but for consistency we could write it as `behavior` here.
(I know this has just been moved so no problem if you don't have time to change this in this PR. :frog: )

Reply:
Done